### ISSUE TITLE
Output encoding error

### ISSUE THREAD

**Issue by mcarpenter** [PayloadSecurity/VxAPI#37]
I'm currently seeing an encoding error from `feed_get_latest` subcommand using the latest commit,  unicode related.

```
$ git log -1
commit 44621bb5a26cfc4412aa91a5373e4d9535e814e7
Author: Jan Miller <jan.miller@payload-security.com>
Date:   Wed Jan 16 07:35:45 2019 +0100

    Update README.md
$ ./vxapi.py feed_get_latest -v

<<< Running 'VxWebService Python API Connector' in version '2.0.1'. Webservice version: 'ProdPatch-HA.?-af7b27f', API version: '2.3.3' >>>


<<< Used API Key >>>

API Key: xxxxxxxx
Auth Level: restricted
User: XXX
<<< Request was sent at 2019-02-04 11:53:49 >>>

Endpoint URL: https://www.hybrid-analysis.com/api/v2/feed/latest
HTTP Method: GET
Sent GET params: {}
Sent POST params: {}
Sent files: {}

<<< Received response at 2019-02-04 11:53:52 >>>

Response status code: 200
Message: Your request was successfully processed by Falcon Sandbox

<<< API query limits for used API Key >>>

Webservice API usage limits: {'minute': 20, 'hour': 200}
Current API usage: {'minute': 2, 'hour': 3}
Is limit reached: No

<<< Showing response >>>


<<< During the code execution, error has occurred. Please try again or contact the support. >>>

Message: ''ascii' codec can't encode characters in position 878906-878907: ordinal not in range(128)'.

Traceback (most recent call last):
  File "./vxapi.py", line 393, in main
    cli_manager.run()
  File "./vxapi.py", line 371, in run
    print(iter_cli_object.get_result_msg())
UnicodeEncodeError: 'ascii' codec can't encode characters in position 878906-878907: ordinal not in range(128)
$
```

**Comment by mcarpenter** [comment]
Relevant SHA256: db5b4a9e96029952242be98030dcd762f92e30c448410b1a964625e03fc97dae. This sample contains extracted file with SHA256 3a8b112c82c3a0b9c74f06cf28f395e9dba94508bef8bf9690551f3d99febe07 and problematic path:

    "file_path": "C:\\MSÐ<9e>CachÐµ\\2.exe",

**Comment by mcarpenter** [comment]
Fix, assumes default UTF-8 output encoding:

``` diff
diff --git a/vxapi.py b/vxapi.py
index 9ce3aaf..665d579 100755
--- a/vxapi.py
+++ b/vxapi.py
@@ -369,7 +369,8 @@ class CliManager:
                     CliMsgPrinter.print_showing_response(arg_iter, current_iteration)
 
                 result = iter_cli_object.get_result_msg()
-                print(result.encode())
+                sys.stdout.buffer.write(result.encode())
+                sys.stdout.write('\n')
 
                 if iter_cli_object.api_object.if_request_success() is False and if_multiple_calls is True and iter_cli_object.api_object.api_expected_data_type == ApiCaller.CONST_EXPECTED_DATA_TYPE_JSON:
                     response_json = iter_cli_object.api_object.get_response_json()
```

**Comment by kkochanski** [comment]
Thanks for your involvement, issue resolved with d192b792e3c3f8312fcd1e0814251d3088f8aec9.
